client/src/data/blankCEJST/index.html

```diff
@@ -14,6 +14,6 @@
 </head>
 
 <body>
-  <!-- your content here... adding comment to trigger build-->
+  <!-- your content here... changing comment to trigger build-->
```

Comment:
Probably not needed

Reply:
The GHA requires some change in the client folder to trigger the build. I first ran it without it. Then placed it in as it wasn't triggered the GHA.